src/engineio/async_drivers/gevent.py

```diff
@@ -3,25 +3,14 @@
 import gevent
 from gevent import queue
 from gevent.event import Event
+from gevent.threading import Thread
 try:
     import geventwebsocket  # noqa
     _websocket_available = True
 except ImportError:
     _websocket_available = False
 
 
-class Thread(gevent.Greenlet):  # pragma: no cover
-    """
-    This wrapper class provides gevent Greenlet interface that is compatible
-    with the standard library's Thread class.
-    """
-    def __init__(self, target, args=[], kwargs={}):
-        super(Thread, self).__init__(target, *args, **kwargs)
-
-    def _run(self):
```

Comment:
This is not part of the `threading.Thread` interface.

Reply:
And why is this a problem? The `Thread` class in this driver is used internally.

Reply:
I see. I couldn't find any occurrences of this method being used anywhere in the codebase and since this is a private method, I decided to not include it anymore. If that's not the case, I'm more than happy to put it back 👍

Reply:
Sorry, my question wasn't directly related to the `_run()` method, I was wondering about the whole change. I don't see any benefit in switching to the `Thread` class instead of using the `Greenlet`  class as I'm doing here. There's nothing wrong with with the `Thread` class as far as I know, but my solution based on `Greenlet` has years of testing, so I'm not sure it makes sense to switch to a different implementation without a good reason.

But in any case, you asked about the `_run()` method. This is where you put the code that runs in the greenlet. You can see how this works in [the documentation](https://www.gevent.org/api/gevent.greenlet.html#gevent.Greenlet.__init__).

Reply:
Thanks for the `_run()` clarification, that makes complete sense then. I will include it in the class, if we decide to go forward with this PR.